File: ProgrammingFundamentals/Java/04.MethodsDebuggingTroubleshootingCode/Lab/src/P01_BlankReceipt.java
public class P01_BlankReceipt {
    public static void main(String[] args) {

        printReceiptHeader();
        printReceiptBody();
        printReceiptFooter();
    }

    static void printReceiptHeader() {
        System.out.println("CASH RECEIPT");
        System.out.println("------------------------------");
    }

    static void printReceiptBody() {
        System.out.println("Charged to____________________");
        System.out.println("Received by___________________");
    }

    static void printReceiptFooter() {
        System.out.println("------------------------------");
        System.out.println("\u00A9 SoftUni");
    }
}

File: ProgrammingFundamentals/C#/04.MethodsDebuggingAndTroubleshootingCode/Lab/P01_BlankReceipt/P01_BlankReceipt.cs
﻿using System;

namespace P01_BlankReceipt
{
    class P01_BlankReceipt
    {
        static void Main(string[] args)
        {
            PrintReceipt();
        }
        static void PrintReceipt()
        {
            PrintReceiptHeader();
            PrintReceiptBody();
            PrintReceiptFooter();
        }

        static void PrintReceiptHeader()
        {
            Console.WriteLine("CASH RECEIPT");
            Console.WriteLine("------------------------------");
        }

        static void PrintReceiptBody()
        {
            Console.WriteLine("Charged to____________________");
            Console.WriteLine("Received by___________________");
        }

        static void PrintReceiptFooter()
        {
            Console.WriteLine("------------------------------");
            Console.WriteLine("\u00A9 SoftUni");
        }
    }
}
